DataFormats/L1TGlobal/interface/GlobalAlgBlk.h

```diff
@@ -109,9 +101,12 @@ class GlobalAlgBlk
     //Prescale Column
     int m_preScColumn;
 
+    //Menu UUID
+    int m_menuUUID;
+    int m_firmwareUUID;
```

Comment:
OK, thanks for the details.

This means we can add a check within the HLT and/or DQM that the menu used in the L1 and the menu from the EventSetup are the same.